ramdump.c

```diff
@@ -61,11 +63,11 @@ int main(int argc, char **argv)
 	if (qdl_debug)
 		print_version();
 
-	ret = qdl_open(&qdl, serial);
+	ret = qdl_open((struct qdl_device *)&qdl, serial);
```

Comment:
Please avoid casting the type, pass &qdl.base as that's what the function expects in your new model.

Reply:
fixed